pkgs/development/python-modules/dash-core-components/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "dash-core-components";
+  version = "1.2.0";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "dash_core_components";
+    sha256 = "910c158081c337c56c6d0a3f617a5bdb275933ffa513ea4ed2448c682650edf6";
+  };
+
+  doCheck = false; # module tests depend on dash, dash depends on this package
+
+  propagatedBuildInputs = [];
```

Comment:
Does this do anything?

Reply:
This module does not specify any packages in its `install_requires` list. It seems rather self-contained (if you ignore the circular dependency between dash and dash-core-components)

Reply:
My point was more that you could remove the whole line

Reply:
Agreed. These have been removed